### ISSUE TITLE
Support tsconfig extends/exclude/include

### ISSUE THREAD

**Issue by ericsampson** [pheuter/svelte-check-rs#18]

**Comment by ericsampson** [comment]
There is literally no code to handle the `include` and `exclude` properties shown below (they are solely deserialized but not handled). 

And the `extends` field is missing entirely.

The only code currently present is handling the compiler_options field.

https://github.com/pheuter/svelte-check-rs/blob/54d938ac8ade5485600a5d857b4a94ae557534a4/crates/svelte-check-rs/src/config.rs#L297-L309

**Comment by pheuter** [comment]
Thanks for the repro! I ran it and can confirm the TypeScript behavior is working as expected.

Your `src/excluded/Broken.svelte` has intentional TS errors:
```typescript
const x: string = 123; // Type error
const y: number = "hello"; // Type error
```

But 0 TypeScript errors are reported from that file - tsgo correctly respects the tsconfig exclude. The output only shows Svelte a11y warnings, no TS errors from the excluded directory.

So to summarize where we're at:
- **TypeScript errors**: ✅ tsconfig `exclude` is respected
- **Svelte diagnostics (a11y)**: tsconfig `exclude` is not applied - as mentioned above, this is currently intentional since tsconfig controls TypeScript checking while `--ignore` controls file discovery for Svelte diagnostics

If you want to exclude paths from both TS and Svelte checks, you can use `--ignore "src/excluded/**"` alongside your tsconfig exclude.

---

Regarding the other issues from your repro:

2. **svelte-ignore pragma not respected** - valid, can you open a separate issue for this?
3. **Missing Svelte checks** - not all Svelte diagnostic checks have been implemented yet, and some won't be by design. For example, `export_let_unused` is a Svelte 4/legacy pattern - svelte-check-rs only targets Svelte 5. For other missing or misbehaving Svelte 5 diagnostic checks, feel free to open separate issues. (Note: CSS checks are lower on my priority list at the moment.)
